code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -327,14 +334,38 @@
 	if (H.internals)
 		H.internals.icon_state = "internal1"
 
-/*
 /datum/species/vox/handle_post_spawn(var/mob/living/carbon/human/H)
-	H.verbs += /mob/living/carbon/human/proc/leap
-	..() */
+	updatespeciescolor(H)
+	H.update_icons()
+	//H.verbs += /mob/living/carbon/human/proc/leap
+	..()
+
+/datum/species/vox/updatespeciescolor(var/mob/living/carbon/human/H) //Handling species-specific skin-tones for the Vox race.
+	if(H.species.name == "Vox") //Making sure we don't break Armalis.
+		switch(H.s_tone)
+			if(4) //Grey Vox.
+				icobase = 'icons/mob/human_races/vox/r_voxgry.dmi'
+				deform = 'icons/mob/human_races/vox/r_def_voxgry.dmi'
+				tail = "voxtail_gry" //Ensures they get an appropriately coloured tail.
+			if(3) //Brown Vox.
+				icobase = 'icons/mob/human_races/vox/r_voxbrn.dmi'
+				deform = 'icons/mob/human_races/vox/r_def_voxbrn.dmi'
+				tail = "voxtail_brn"
+			if(2) //Dark Green Vox.
+				icobase = 'icons/mob/human_races/vox/r_voxdgrn.dmi'
+				deform = 'icons/mob/human_races/vox/r_def_voxdgrn.dmi'
+				tail = "voxtail_dgrn"
+			else  //Default Green Vox.
+				icobase = 'icons/mob/human_races/vox/r_vox.dmi'
+				deform = 'icons/mob/human_races/vox/r_def_vox.dmi'
+				tail = "voxtail"
+
+		H.update_dna()
 
 /datum/species/vox/armalis/handle_post_spawn(var/mob/living/carbon/human/H)
+	H.verbs += /mob/living/carbon/human/proc/leap
 	H.verbs += /mob/living/carbon/human/proc/gut
-	..()
+	//..()
```

Comment:
I wasn't around when Vox lost their leap, but I imagine Armalis only lost it because them having leap relied upon their parent call and since it was just commented out during the voxleap removal, they lost it too

But yeah, I only ever see Armalis in adminbus and even then, like you said, basically unused

I didn't look so far back as the grant_abilities call, thanks for catching this